packages/berry-cli/sources/cli.ts

```diff
@@ -38,6 +39,25 @@ function runBinary(path: PortablePath) {
   }
 }
 
+async function printAvailableScripts(configuration: Configuration) {
+  const stdout = process.stdout;
+  const report = await StreamReport.start({configuration, stdout}, async (report: StreamReport) => {
+    const {workspace} = await Project.find(configuration, configuration.startingCwd);
+    if (!workspace)
+      throw new WorkspaceRequiredError(configuration.startingCwd);
+
+
+    var projectCommands: string[] = []
+    workspace!.manifest.scripts.forEach((value: string, key: string) => {
+      projectCommands.push(`${' '}- ${key}`)
+      projectCommands.push(`  ${' '} ${value}`)
+    });
+
+    report.reportInfo(null, `Project commands\r\n${projectCommands.join('\r\n')}`);
```

Comment:
Prefer calling `reportInfo` once for each entry.

Reply:
I implemented the output format similar to `config` command 👍